docs/design-proposals/multi-tenancy/multi-tenancy-overview.md

```diff
@@ -0,0 +1,149 @@
+# Arktos Multi-Tenancy Overview
+
+Xiaoning Ding, Qian Chen, Peng Du, Hong Chen
+
+## Introduction
+
+This document gives an overview of the multi-tenancy design in Arktos, including its motivation, requirements, design overview and lists of components changes and execution plans. For the detailed design of each part, please refer to the other documents in the same document directory.
+
+## Motivation
+
+Kubernetes provides a solution for automating deployment, scaling, and operations of application containers across clusters of hosts. However, it does not have support multi-tenancy. There is no explicit “tenant” concept in the system, and there is no resource isolation, usage isolation or performance isolation for different tenants.
```

Comment:
minor grammar issue : change "not have support multi-tenancy" to "not  support multi-tenancy" .

Reply:
Yeah, will change.